quadriga/src/test/java/edu/asu/diging/quadriga/core/literals/TestLiterals.java

```diff
@@ -0,0 +1,18 @@
+package edu.asu.diging.quadriga.core.literals;
+
+public class TestLiterals {
+    public static final String BLANK = "";
+
+    public static final class CollectionLiterals {
+        public static final String COLLECTION_NAME = "Collection name";
+        public static final String COLLECTION_DESC = "Collection description";
+        public static final String EDITED_NAME = "Edited name";
+        public static final String EDITED_DESC = "Edited description";
```

Comment:
These should just be constants in the test class since we won't need them in any other test classes most likely.